common/djangoapps/adaptive_learning/config/tests/test_models.py

```diff
@@ -27,10 +27,10 @@ def setUp(self):
               (False, True, False),
               (True, True, True))
     @ddt.unpack
-    def test_method_feature_enabled(self, global_enabled_flag, course_enabled_flag, should_course_be_enabled):
+    def test_method_feature_enabled(self, global_enabled_flag, course_enabled_flag, feature_enabled_for_course):
```

Comment:
@tomaszgy Sorry for not catching this earlier, but I think it would be OK to just call this `test_feature_enabled`. People reading this code should know that it's a method :)

Reply:
👍